llvm/test/CodeGen/SPIRV/hlsl-resources/BufferStore.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: llc -O0 -verify-machineinstrs -mtriple=spirv-vulkan-library %s -o - | FileCheck %s
```

Comment:
Shall we enable optimizations? Input is from an external function, output should not be optimzed away, and thus we could check passes are fine with what we do, and verify our intrinsics don't get optimized away?